projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+# School events
+
+School events are events that are specifically organized and intented for:
+
+* students (e.g. a school performance)
+* teachers (e.g. an after-school training)
+
+School events are published on [Cultuurkuur](https://www.cultuurkuur.be), but not on UiTinVlaanderen.
+
+> If you’re planning to create school events in UiTdatabank with the Entry API, please contact <content.cultuurkuur@publiq.be> in advance so that we can guide you through this process.
+
+## Types
+
+We distinguish 3 different types of school events:
+
+1. **School performances** are events of which both the date and the location is known in advance.
+2. **Guided tours** are events that have no specific date, but they do have a location.
+3. **Bookable events** are events that have no specific date and location (or the date and location is not known in advance). Both the date and the location are determined in mutual agreement between the organizer or artist and the consumer (school).
+
+### School performances
+
+School performances are events of which both the date and the location is known in advance. For example, a theater performance aimed at a toddler of 3-4 years old in hetpaleis on 14/05/2023, 14:30 - 16:00 PM.
+
+* ✅ date is known in advance
+* ✅ location is known in advance
+
+Since school performances both have a date and a location, you can create them in a very similar way to regular events:
+
+* you can use a calendarType of your preference (`single`, `multiple` or `periodic`)
+* for the location you must use the URL of an existing place as `location.@id` in the `POST /events` request of the event
+
+### Guided tours
+
+Guided tours are events that have no specific date (or the date is not known in advance), but they do have a location. For example, a guided tour in the Royal Museum of Fine Arts Antwerp.
+
+* ❌ date is not known in advance
+* ✅ location is known in advance
+
+In order to create a guided school tour you must use:
+
+* calendarType `permanent`
+* the URL of an existing place as `location.@id` in the `POST /events` request of the event
+
+### Bookable events
+
+Bookable events are events that have no specific date and location (or the date and location is not known in advance). Both the date and the location are determined in mutual agreement between the organizer or artist and the consumer (school).
+
+For example, as a school you can book Stijn Meuris for a school performance at your school or local cultural centre, on a date that suits both the artist and the school.
+
+* ❌ date is not known in advance
+* ❌ location is not known in advance
+
+In order to create a bookable school event you must use
+
+* calendarType `permanent`
+* the URL of the dummy place as `location.@id` in the `POST /events` request of the bookable events
```

Comment:
This is called a "location in consultation with the school" in the terminology page. https://docs.publiq.be/docs/uitdatabank/76b7bd22f7f51-terminology#location-in-consultation-with-the-school

We can definitely rename that (I just based it on the "locatie in overleg met school"), but I would not use the name "dummy location" for this because we also use that for locations that have no `@id`: https://docs.publiq.be/docs/uitdatabank/76b7bd22f7f51-terminology#dummy-location

(Other occurrences of "dummy location" in this guide about school events should also be renamed if you agree)

Reply:
👍 Changed it to "location in consultation with the school"